### ISSUE TITLE
Excel export not working

### ISSUE THREAD

**Issue by PGjedde** [LCA-ActivityBrowser/activity-browser#951]
When trying to export an excel file of a new database i get the following error message:

> Traceback (most recent call last):
>   File "C:\Users\my_username\anaconda3\envs\ab\lib\site-packages\xlsxwriter\worksheet.py", line 523, in _write
>     f = float(token)
> TypeError: float() argument must be a string or a number, not 'dict'
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "C:\Users\my_username\anaconda3\envs\ab\lib\site-packages\activity_browser\ui\wizards\db_export_wizard.py", line 38, in accept
>     self.perform_export()
>   File "C:\Users\my_username\anaconda3\envs\ab\lib\site-packages\activity_browser\ui\wizards\db_export_wizard.py", line 52, in perform_export
>     EXPORTERS[export_as](db_name, out_path)
>   File "C:\Users\my_username\anaconda3\envs\ab\lib\site-packages\activity_browser\bwutils\exporters.py", line 103, in write_lci_excel
>     sheet.write(row_index, col_index, value, frmt(value))
>   File "C:\Users\my_username\anaconda3\envs\ab\lib\site-packages\xlsxwriter\worksheet.py", line 90, in cell_wrapper
>     return method(self, *args, **kwargs)
>   File "C:\Users\my_username\anaconda3\envs\ab\lib\site-packages\xlsxwriter\worksheet.py", line 456, in write
>     return self._write(row, col, *args)
>   File "C:\Users\my_username\anaconda3\envs\ab\lib\site-packages\xlsxwriter\worksheet.py", line 528, in _write
>     raise TypeError("Unsupported type %s in write()" % type(token))
> TypeError: Unsupported type <class 'dict'> in write()

In my current installation of activity browser I followed the [installation guide for Ecoinvent <=3.8 (new installation)](https://github.com/LCA-ActivityBrowser/activity-browser#ecoinvent-38-new-installation), and so I have the following installed in my conda environment (naturally there is more, but I think this is what is relevant to know?):

Name          -          Version         
activity-browser -         2.7.4  
brightway2          -      2.4.3    
bw2analyzer         -      0.10    
bw2calc                -   1.8.2      
bw2data                -   3.6.6     
bw2io                 -    0.8.7       
bw2parameters   -          0.7
bw_migrations      -       0.2

**Comment by Zoophobus** [comment]
Hi,

It's weird that it detects a dictionary within the database, I really  can't tell you why that might be happening. It's also a little frustrating as I need to replicate this but don't know how, which brings me to my question, would it be possible for you to share your database?

best,
Jon

**Comment by PGjedde** [comment]
Hi @Zoophobus, thank you for reaching back quickly. I was trying to export a database with inventories I set up myself. My foreground invetories were connected to the EcoInvent 3.8 APOS. I noticed that I at some point had copied in an inventory from that database to my own. After deleting it I was able to export my database. And so the issue has been fixed now for my case. If you have an idea to why this caused the dict problem though I'd be interested to learn. Thank you :)

Best,
Philip
